### PR TITLE
Login: Add a backup codes screen

### PR DESCRIPTION
**Pull request by scruffian**
This pull request adds a new option to use backup codes for 2FA if the customer has lost their phone:
screen_shot_2017-05-11_at_10 35 08
screen_shot_2017-05-11_at_10 37 41
  
#### Testing instructions
  
1. Run `git checkout add/backup-codes` and start your server, or open a [live branch](https://calypso.live/?branch=add/backup-codes)
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Enter a username and password
4. Check that on the 2FA screens you see an "I can't access my phone link" in the footer
5. Check that the "I can't access my phone" link takes you to a page to enter you backup code.
  
#### Reviews
  
- [ ] Code
- [ ] Product

   
@Automattic/sdev-feed

### CODE REVIEW

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx [hunk 1]
```diff
@@ -74,10 +74,17 @@ class VerificationCodeForm extends Component {
...
 		let helpText = translate( 'Type in the code generated by your Authenticator mobile application.' );
+		let labelText = translate( 'Verification Code' );
 		if ( twoFactorAuthType === 'sms' ) {
 			helpText = translate( 'A text message with the verification code was just sent to your phone number.' );
 		}
 
+		if ( twoFactorAuthType === 'backup' ) {
+			helpText = translate( "If you can't access your phone enter one of the 10 backup codes that were provided " +
+			'when you set up two-step authentication to continue.' );
+			labelText = translate( 'Backup code' );
```
Comment (a8ci18n) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Backup Codes' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx [hunk 2]
```diff
@@ -74,10 +74,24 @@ class VerificationCodeForm extends Component {
...
+		let labelText = translate( 'Verification Code' );
+		let smallPrint;
 		if ( twoFactorAuthType === 'sms' ) {
 			helpText = translate( 'A text message with the verification code was just sent to your phone number.' );
 		}
 
+		if ( twoFactorAuthType === 'backup' ) {
+			helpText = translate( "If you can't access your phone enter one of the 10 backup codes that were provided " +
+				'when you set up two-step authentication to continue.' );
+			labelText = translate( 'Backup code' );
```
Comment (a8ci18n) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Backup Codes' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx [hunk 3]
```diff
@@ -74,10 +74,24 @@ class VerificationCodeForm extends Component {
...
+			smallPrint = (
+				<div className="two-factor-authentication__small-print">
+					{ translate( 'If you lose your device, accidentally remove the authenticator app, or are otherwise ' +
+						'locked out of your account, the only way to get back in to your account is by using a Backup Code.' ) }
```
Comment (drewblaisdell) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: I think we probably don't want to capitalize `Backup Code` (we don't capitalize `verification code` in the other message).

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx (hunk 2 above)
Comment (drewblaisdell) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: We probably want to match the case of the other label text with `Backup Code`.

**Comment by drewblaisdell** [comment]
I had some a couple really minor notes, and cherry-picked a commit from a different PR that is related to the copy here. I also tested this and it works (there isn't much to test as backup codes use the same input/endpoint but I tested one of them). If you're cool with my changes, feel free to merge!

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx [hunk 4]
```diff
@@ -74,10 +74,24 @@ class VerificationCodeForm extends Component {
...
+		let labelText = translate( 'Verification Code' );
+		let smallPrint;
 		if ( twoFactorAuthType === 'sms' ) {
 			helpText = translate( 'A text message with the verification code was just sent to your phone number.' );
 		}
 
+		if ( twoFactorAuthType === 'backup' ) {
+			helpText = translate( "If you can't access your phone enter one of the 10 backup codes that were provided " +
+				'when you set up two-step authentication to continue.' );
+			labelText = translate( 'Backup Code' );
```
Comment (a8ci18n) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Backup Codes' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**